### PR TITLE
UpdateServiceDsc: Update localization strings

### PR DESCRIPTION
**Pull request by johlju**
#### Pull Request (PR) description
- UpdateServicesApprovalRule
  - Updated to load localization strings correctly.
- UpdateServicesCleanup
  - Updated to load localization strings correctly.
- UpdateServicesServer
  - Updated to load localization strings correctly.

#### This Pull Request (PR) fixes the following issues
Blocking PR's.

#### Task list

- [x] Added an entry under the Unreleased section of the change log in the CHANGELOG.md.
      Entry should say what was changed, and how that affects users (if applicable).
- [ ] Resource documentation added/updated in README.md.
- [ ] Resource parameter descriptions added/updated in README.md, schema.mof
      and comment-based help.
- [ ] Comment-based help added/updated.
- [ ] Localization strings added/updated in all localization files as appropriate.
- [ ] Examples appropriately added/updated.
- [ ] Unit tests added/updated. See [DSC Resource Testing Guidelines](https://github.com/PowerShell/DscResources/blob/master/TestsGuidelines.md).
- [ ] Integration tests added/updated (where possible). See [DSC Resource Testing Guidelines](https://github.com/PowerShell/DscResources/blob/master/TestsGuidelines.md).
- [x] New/changed code adheres to [DSC Resource Style Guidelines](https://github.com/PowerShell/DscResources/blob/master/StyleGuidelines.md) and [Best Practices](https://github.com/PowerShell/DscResources/blob/master/BestPractices.md).

- - -
This change is [Reviewable](https://reviewable.io/reviews/dsccommunity/UpdateServicesDsc/79)

### CODE REVIEW

**Review by Borgquite — COMMENTED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/dsccommunity/UpdateServicesDsc/79#-:-NzxytbS4PCRY0doIWkv:bnfp4nl)* status: 0 of 4 files reviewed, all discussions resolved

**Review by Borgquite — COMMENTED** [review]
> Reviewed 4 of 4 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/dsccommunity/UpdateServicesDsc/79)* status: <span>:shipit:</span> complete! all files reviewed, all discussions resolved (waiting on @johlju)

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,25 +7,31 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- Added required modules, Sampler.GitHubTasks, powershell-yaml
 - Added wildcard support in Products parameter of UpdatesServicesServer resource.
  (issue #13)
 
 ### Fixed
 
 - Fix deploy job in AzurePipeline, Added Sampler.GithubTasks in build.yaml
-- Fix issue #61 and #67, with add a foreach loop when `Set-TargetRessource` found 
+- Fix issue #61 and #67, with add a foreach loop when `Set-TargetResource` found
 multiple products dor the same `Title`.
```
Comment (Borgquite) [inline CHANGELOG.md]: Did spot one extra error here ('dor' -> 'for') but didn't know if flagging it here would hold up the merge. I can fix in a later PR if necessary.

**Comment by johlju** [comment]
Will fix those tomorrow. Not at a computer right now so can't merge anyway.

File: CHANGELOG.md (hunk 1 above)
Comment (johlju) [reply CHANGELOG.md]: Done

**Pull request merged**